.github/workflows/main.yml

```diff
@@ -0,0 +1,35 @@
+name: Build docker image
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: brpaz/hadolint-action@v1.2.1
+        name: Lint
+        with:
+          dockerfile: Dockerfile
+      -
+        name: Login to GitHub Container Registry
+        uses: docker/login-action@v1 
+        with:
+          registry: ghcr.io
```

Comment:
I think we can stick with the default Docker hub for now as this doesn't require an additional step of `docker login`